hil-test/src/bin/twai.rs

```diff
@@ -59,3 +68,93 @@ mod tests {
         assert_eq!(frame.data(), &[1, 2, 3])
     }
 }
+
+#[embedded_test::tests(default_timeout = 3, executor = hil_test::Executor::new())]
+mod async_tests {
```

Comment:
TIL you can have multiple test modules. Wow 🤯

Reply:
Haha, I didn’t know it either, but I gave it a try with very little hope 😄